crates/bevy_render/src/view/window.rs

```diff
@@ -54,11 +55,12 @@ pub struct ExtractedWindow {
 
 #[derive(Default, Resource)]
 pub struct ExtractedWindows {
-    pub windows: HashMap<WindowId, ExtractedWindow>,
+    pub primary: Option<Entity>,
+    pub windows: HashMap<Entity, ExtractedWindow>,
```

Comment:
I think I'd prefer if this were a future work item, otherwise this PR will just end up getting more bloated since it'd be easier to introduce small logical errors in the porting of the code to entities/components

Reply:
Agreed. This currently works and I'd like to get this merged asap.